serving/src/main/java/ai/djl/serving/http/InferenceRequestHandler.java

```diff
@@ -56,6 +56,7 @@ public class InferenceRequestHandler extends HttpRequestHandler {
 
     private static final Pattern PATTERN =
```

Comment:
Can we update this variable name to be more clear now that we're introducing INVOKE_PATTERN? Or is it possible to combine these into a single pattern?

Reply:
done